docs/graphql.md

```diff
@@ -0,0 +1,68 @@
+# Using GraphQL
+
+\_Note: If testing this locally, you must have telescope running locally as well. Please refer to the documentation [here](./environment-setup.md)
+
+# Adding/Modifying Resolvers for GraphQL
+
+1. Our resolvers can be found in [here](../src/backend/web/graphql/index.js) under module.exports.resolvers.
```

Comment:
"can be found in [src/backend/web/graphql/index.js](../src/backend/web/graphql/index.js)"